app/api/leaderboard/functions.ts

```diff
@@ -28,7 +28,7 @@ export const getLeaderboardData = async (
     )
     .sort((a, b) => {
       if (sortBy === "pr_stale") {
-        return b.activityData.pr_stale - a.activityData.pr_stale;
+        return (b.activityData.pr_stale ?? 0) - (a.activityData.pr_stale ?? 0);
```

Comment:
Why was this unrelated line changed? Could you specify the issue that's being fixed by this change?